File: CodeChef/april15/interview.cpp
#include <bits/stdc++.h>
using namespace std;

#define lld long long int

int main()
{
    ios_base::sync_with_stdio(false);
    lld t;
    scanf("%lld",&t);
    while(t--)
    {
        lld n;
        lld sum;
        scanf("%lld",&n);

        if(n != 1)
        {
            sum=n+1;
            lld endloop;
            double limit = sqrt(n);
            if(limit%1 == 0)
            {   
                sum += (long)limit;
                    endloop = (long)(limit-1);
                }
                else
                    endloop=(long)limit;

                for(lld i=2; i<=endloop; i++)
                {
                    if(n%i == 0)
                    {
                        long temp = n/i;
                        sum += (temp + (long)(n/temp));
                    }
                }
        }
        else
            sum=1;
        printf("%lld\n", sum);
    }

    return 0;
}

File: CodeChef/april15/interview.py
t = int(raw_input())
while t>0:
	t = t - 1
	su = 0
	n = int(raw_input())
	if n != 1:
		su = n + 1
		temp = z = 0
		l = 0.0
		l = n**.5
		if l %1 == 0:
			su = su + long(l)
			z = long(l - 1)
		else:
			z = long(l)
		i = 2
		while i<=z:
			if n % 1 == 0:
				temp = n/i
				su = su + temp + long(n/temp)
			i += 1
	else:
		su = 1
	print su